corehq/apps/registration/static/registration/js/password_reset_form.js

```diff
@@ -0,0 +1,4 @@
+hqDefine('registration/js/password_reset_form', [
```

Comment:
I suspect that this should also contain the draconian password enforcement, but understand if you don't want to deal with changing that behavior in this PR.